packages/cloudvision-connector/src/utils.ts

```diff
@@ -20,53 +20,51 @@ import { fromByteArray, toByteArray } from 'base64-js';
 import MurmurHash3 from 'imurmurhash';
 
 import {
-  NotifCallback,
-  PlainObject,
-  SubscriptionIdentifier,
-  PublishCallback,
-  RequestArgs,
-} from '../types';
-import {
+  CloseParams,
   CloudVisionBatchedNotifications,
   CloudVisionBatchedResult,
   CloudVisionDatasets,
   CloudVisionNotifs,
+  CloudVisionParams,
+  CloudVisionPublishRequest,
   CloudVisionResult,
   CloudVisionServiceResult,
   CloudVisionStatus,
-} from '../types/notifications';
-import {
-  CloseParams,
-  CloudVisionParams,
+  NotifCallback,
+  Options,
+  PublishCallback,
   Query,
+  RequestArgs,
   SearchOptions,
   SearchType,
+  ServiceRequest,
+  SubscriptionIdentifier,
   WsCommand,
-} from '../types/params';
-import { Options, CloudVisionPublishRequest, ServiceRequest } from '../types/query';
+} from '../types';
+import { PlainObject } from '../types/utility';
```

Comment:
any reason why you aren't importing this one from the root?

Reply:
I figured we didn't want to export `PlainObject` from the package, similar to the emitter types.
I can add it.

Reply:
Actually, it turns out we don't even need this type at all.